src/jdk.crypto.ec/share/classes/sun/security/ec/XECParameters.java

```diff
@@ -42,7 +42,7 @@ public class XECParameters {
     static final XECParameters X25519;
     static final XECParameters X448;
 
-    static ParametersMap<XECParameters> namedParams = new ParametersMap<>();
+    private static final ParametersMap<XECParameters> namedParams = new ParametersMap<>();
```

Comment:
Small code style comment - we try to keep our code lines to 80 characters or less as it avoids scrolling for things like split screen code reviews. Suggest putting a line break after `=` and indenting the next line four spaces, ex:

```
    private static final ParametersMap<XECParameters> namedParams =
        new ParametersMap<>();
```
Same comment below.